homeassistant/components/forked_daapd/media_player.py

```diff
@@ -63,6 +66,9 @@
     TTS_TIMEOUT,
 )
 
+if TYPE_CHECKING:
+    from homeassistant.components.media_player import BrowseMedia
+
```

Comment:
I think it was decided to avoid the `if TYPE_CHECKING` for imports, unless there is a compelling reason to do so.
In this case, you are already importing the `homeassistant.components.media_player` module for `MediaPlayerEntity` so there is not benefit.

Reply:
Whoops, thanks.
Regarding the broader` TYPE_CHECKING` part, we just import regardless of whether it's actually used or not?

Reply:
My understanding if that unless it causes a real performance hit, then we don't guard.
See this discussion: #75983

Reply:
In this case since we're importing the file anyway, there's no benefit in using TYPE_CHECKING. However, for the more general case, my reading of that discussion is that we don't want to make PRs solely for guarding imports with TYPE_CHECKING, otherwise we would get a bunch of extra PRs with little benefit. However, there doesn't seem to be a strong case for not adding such imports incidentally as part of another PR. Do you think it really adds extra load on the contributor/reviewer?